### ISSUE TITLE
Shift|Option|Cmd + click combinations work unstable

### ISSUE THREAD

**Issue by DenKoren** [tekezo/Karabiner-Elements#204]
After #5 close problem still randomly appears during day.
Reboot helps to solve the problem temporary, but it still exists and appears unpredictable.

I created new issue since #5 is closed and you can just miss new comments in it.

P.S.:
Thanks for your work, Karabiner is a great app and I hope this one will be even better soon.

**Comment by andyph666** [comment]
I have this problem as well. When I exit karabiner-elements, shift+click and cmd+click works just fine.

**Comment by aboualiaa** [comment]
I also had this problem. Some how restarting Karabiner solves it (temporarily)

**Comment by dhleong** [comment]
My issue may be related. Usually after waking the mac from sleep (but not every time), using gestures with Better Touch Tool to send key strokes will send a shift along with them (so instead of apple+w to close a tab, I get shift+apple+w, which closes the whole window). I contacted the BTT author and he believes it's not on his end (which I tend to agree with since restarting BTT does not fix it). Oddly, the shift key is not just *held*, since I can use the keyboard shortcuts from the keyboard just fine.

Quitting and re-opening karabiner-elements appears to have fixed the issue just now.

Happy to open a separate ticket if this seems unrelated to the maintainers.

**Comment by cadavre** [comment]
Same here. Shift|Cmd + click stops working. Karabiner restart helps – after exit Shift|Cmd works fine and still works when turning Karabiner on again.

**Comment by aortan** [comment]
Using the latest version and also having issues with screen zoom (CTRL + scroll up) not working when the app is running.

P.s. Appreciate your work. Cant live without this app :)

**Comment by DenKoren** [comment]
After one of updates the problem dissapeared. I can't remember how long ago I restarted karabiner-elements, so it definitely became much more frequent, or have been totally solved.

Thanks!
